### PR TITLE
Some additions

### PR DESCRIPTION
**Pull request by Pe46dro**
From issues:
#301 
#316 
#586 
#602 
#800 

My PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

I don't know why it show my already merged commits.

### CODE REVIEW

**Comment by nodiscc** [comment]
I think API management should be moved as a subsection of Software Development

File: README.md [hunk 1]
```diff
@@ -268,6 +278,7 @@ _Mailing lists servers and mass mailing software - one message to many recipient
   * [Dada Mail](http://dadamailproject.com/) - A web-based list management system that can be used for announcement lists and/or discussion lists. ([Source Code](https://github.com/justingit/dada-mail)) `GPLv2` `Perl`
   * [Mailman](https://www.gnu.org/software/mailman/) - The Gnu mailing list server. `GPL` `Python`
   * [Mailtrain](https://mailtrain.org/) - self hosted newsletter application built on Node.js (v5+) and MySQL (v5.5+ or MariaDB). ([Source Code](https://github.com/andris9/mailtrain)) `GPLv3` `Nodejs`
+  * [MailyHerald](http://mailyherald.org/) - Helps you send and manage your application mailings. Think of Maily as a self-hosted Mailchimp alternative you can easily integrate into your site. MailyHerald is great both for email marketing and conducting the daily stream of notifications you send to your users.. ([Source Code](https://github.com/Sology/maily_herald)) `LGPLv3` `Ruby`
```
Comment (nodiscc) [inline README.md]: Double dot.

File: README.md [hunk 2]
```diff
@@ -520,6 +532,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
   * [Git Annex](http://git-annex.branchable.com) - File synchronization between computers, servers, external drives. ([Source Code](https://github.com/joeyh/git-annex)) `GPLv3` `Haskell`
   * [Kinto](https://kinto.readthedocs.org) - Kinto is a minimalist JSON storage service with synchronisation and sharing abilities. ([Source Code](https://github.com/Kinto)) `Apache` `Python`
   * [Nextcloud](https://nextcloud.com/) - Access & share your files, calendars, contacts, mail & more from any device, on your terms. ([Source Code](https://github.com/nextcloud/server)) `MIT` `PHP`
+  * [OpenMediaVault](http://www.openmediavault.org/) - OpenMediaVault is the next generation network attached storage (NAS) solution based on Debian Linux. It contains services like SSH, (S)FTP, SMB/CIFS, DAAP media server, RSync, BitTorrent client and many more.. ([Source Code](https://github.com/openmediavault/openmediavault)) `GPLv3` `PHP`
```
Comment (nodiscc) [inline README.md]: Double dot.

File: README.md [hunk 3]
```diff
@@ -1071,6 +1082,7 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
   * [Puffin](http://puffin.rocks/) - Lightweight webapp catalog based on containers, with user interface à la mobile app store. ([Demo](http://puffin.rocks/), [Source Code](https://github.com/puffinrocks/puffin)) `AGPLv3` `Python/Docker`
   * [Sandstorm](https://sandstorm.io/) - Personal server for running self-hosted apps easily and securely. ([Demo](https://demo.sandstorm.io/), [Source Code](https://github.com/sandstorm-io/sandstorm)) `Apache` `C++/Other`
   * [sovereign](https://github.com/sovereign/sovereign) - A set of Ansible playbooks to build and maintain your own private cloud: email, calendar, contacts, file sync, IRC bouncer, VPN, and more. `GPLv3` `YAML/Other`
+  * [Syncloud](http://syncloud.org/) - Your own online file storage, social network or email server.. ([Source Code](https://github.com/syncloud/platform)) `GPLv3/Other` `Python/Other`
```
Comment (nodiscc) [inline README.md]: Double dot.

File: README.md [hunk 4]
```diff
@@ -69,6 +69,7 @@ See [Contributing](#contributing).
     - [Bug Trackers](#bug-trackers)
     - [IDE/Tools](#idetools)
     - [Continuous Integration](#continuous-integration)
+	- [API Management](#api-management)
```
Comment (nodiscc) [inline README.md]: Please remove the table of contents entry

File: README.md [hunk 5]
```diff
@@ -109,8 +110,8 @@ _Business Intelligence_
   * [Metabase](http://www.metabase.com/) - Simple Dashboarding and GUI Query tool, Nightly Emails and Slack Integration w/ PostgreSQL, MySQL, Redshift and other DBs. ([Source Code](https://github.com/metabase/metabase)) `AGPLv3` `Clojure`
   * [Redash](http://redash.io) - connect to over 18 types of databases (SQL and "NoSQL"), query your data, visualize it and create dashboards. Everything has a URL that can be shared. Slack and HipChat integration. ([Demo](https://demo.redash.io), [Source Code](https://github.com/getredash/redash)) `BSD` `Python`
 
-
-## Automation
+ 
+ ## Automation
```
Comment (nodiscc) [inline README.md]: Please remove the added spaces

File: README.md [hunk 6]
```diff
@@ -520,6 +523,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
   * [Git Annex](http://git-annex.branchable.com) - File synchronization between computers, servers, external drives. ([Source Code](https://github.com/joeyh/git-annex)) `GPLv3` `Haskell`
   * [Kinto](https://kinto.readthedocs.org) - Kinto is a minimalist JSON storage service with synchronisation and sharing abilities. ([Source Code](https://github.com/Kinto)) `Apache` `Python`
   * [Nextcloud](https://nextcloud.com/) - Access & share your files, calendars, contacts, mail & more from any device, on your terms. ([Source Code](https://github.com/nextcloud/server)) `MIT` `PHP`
+  * [OpenMediaVault](http://www.openmediavault.org/) - OpenMediaVault is the next generation network attached storage (NAS) solution based on Debian Linux. It contains services like SSH, (S)FTP, SMB/CIFS, DAAP media server, RSync, BitTorrent client and many more. ([Source Code](https://github.com/openmediavault/openmediavault)) `GPLv3` `PHP`
```
Comment (nodiscc) [inline README.md]: Please move this to https://github.com/Kickball/awesome-selfhosted#self-hosting-solutions (it's a distribution)

File: README.md [hunk 7]
```diff
@@ -268,6 +269,7 @@ _Mailing lists servers and mass mailing software - one message to many recipient
   * [Dada Mail](http://dadamailproject.com/) - A web-based list management system that can be used for announcement lists and/or discussion lists. ([Source Code](https://github.com/justingit/dada-mail)) `GPLv2` `Perl`
   * [Mailman](https://www.gnu.org/software/mailman/) - The Gnu mailing list server. `GPL` `Python`
   * [Mailtrain](https://mailtrain.org/) - self hosted newsletter application built on Node.js (v5+) and MySQL (v5.5+ or MariaDB). ([Source Code](https://github.com/andris9/mailtrain)) `GPLv3` `Nodejs`
+  * [MailyHerald](http://mailyherald.org/) - Helps you send and manage your application mailings. Think of Maily as a self-hosted Mailchimp alternative you can easily integrate into your site. MailyHerald is great both for email marketing and conducting the daily stream of notifications you send to your users. ([Source Code](https://github.com/Sology/maily_herald)) `LGPLv3` `Ruby`
```
Comment (AndyR207) [inline README.md]: The description here is too long. 250 characters max.

File: README.md [hunk 8]
```diff
@@ -268,6 +269,7 @@ _Mailing lists servers and mass mailing software - one message to many recipient
   * [Dada Mail](http://dadamailproject.com/) - A web-based list management system that can be used for announcement lists and/or discussion lists. ([Source Code](https://github.com/justingit/dada-mail)) `GPLv2` `Perl`
   * [Mailman](https://www.gnu.org/software/mailman/) - The Gnu mailing list server. `GPL` `Python`
   * [Mailtrain](https://mailtrain.org/) - self hosted newsletter application built on Node.js (v5+) and MySQL (v5.5+ or MariaDB). ([Source Code](https://github.com/andris9/mailtrain)) `GPLv3` `Nodejs`
+  * [MailyHerald](http://mailyherald.org/) - Maily as a self-hosted Mailchimp® you can easily integrate with your site. Helps you send and manage your application mailings. It support email marketing and conducting the daily stream of notifications you send to your users. ([Source Code](https://github.com/Sology/maily_herald)) `LGPLv3` `Ruby`
```
Comment (Kickball) [inline README.md]: Please change as to is at the start of the first sentence and remove the icon after MailChimp.

**Pull request merged**